packages/react-charting/etc/react-charting.api.md

```diff
@@ -679,7 +679,11 @@ export interface IHorizontalBarChartProps {
     chartDataMode?: ChartDataMode;
     className?: string;
     culture?: string;
+    customEmptyMsg?: string;
```

Comment:
create a new interface having all props for error handling

Reply:
done